models/iosxr/lacp/iosxr_lacp.yaml

```diff
@@ -0,0 +1,54 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: iosxr
+RESOURCE: lacp
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: iosxr_lacp
+  version_added: 2.9
+  short_description: Manage Global Link Aggregation Control Protocol (LACP) on IOS-XR devices.
+  description: 
+    - This module manages Global Link Aggregation Control Protocol (LACP) on IOS-XR devices.
+  author: Nilashish Chakraborty (@nilashishc)
+  options:
+    config:
+      description: The provided configurations.
+      type: dict
+      suboptions:
+        system:
+          description: This option sets the default system parameters for LACP bundles.
+          type: dict
+          suboptions:
+            priority:
+              description:
+                - The system priority to use in LACP negotiations.
+                - Lower value is higher priority.
+                - Refer to vendor documentation for valid values.
+              type: int
+            mac:
```

Comment:
@NilashishC @cidrblock For nxos mac has two options. address followed by the role.
so the model for nxos looks like `mac: {address: '', role: ''}`

Reply:
@trishnaguha I have updated that in https://github.com/ansible/ansible/pull/59596